docs/src/useReducedMotion.doc.js

```diff
@@ -0,0 +1,62 @@
+// @flow strict
+import * as React from 'react';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="useReducedMotion"
+    description={`
+      \`useReducedMotion\` uses the <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion" target="_blank">prefers-reduced-motion</a> CSS media feature to detect
+      if the user has requested that the system minimize the amount of non-essential motion.
```

Comment:
Can we provide more guidance here on when devs should use this? Not everyone will click the link for a full explanation

Reply:
@jennyscript added - mind taking another look?